### ISSUE TITLE
sails.io.js: Ship with minified script

### ISSUE THREAD

**Issue by sailsbot** [balderdashy/sails#4050]
_From @l1br3 on February 27, 2017 15:58_

Hi, as the title says it would be very nice to have a minified version of the script in the bower branch so I can directly import it in my html.

EDIT: 
I know that sails usually minify/uglyfy scripts with grunt but I'm using webpack and disabled grunt.
As webpack is becoming one of the most used solution, I think that it would be valuable to make this modification.

thanks for your job

_Copied from original issue: balderdashy/sails.io.js#117_

**Comment by sailsbot** [comment]
@l1br3 Thanks for posting, we'll take a look as soon as possible.  In the meantime, if you haven’t already, please carefully read the [issue contribution guidelines](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md#ii-issue-contributions) and double-check for any missing information above.  In particular, please ensure that this issue is about a stability or performance bug with a  documented feature; and make sure you’ve included detailed instructions on how to reproduce the bug from a clean install.   Finally, don’t forget to include the version of Node.js you tested with, as well as your version of Sails or Waterline, and of any relevant standalone adapters/generators/hooks.

Thank you!

**Comment by sailsbot** [comment]
_From @mikermcneil on March 21, 2017 15:11_

@l1br3 Thanks for the reminder.  I'm game!  There's an old PR here-- looks like it could probably be updated: https://github.com/balderdashy/sails.io.js/pull/67/files

**Comment by mikermcneil** [comment]
(@l1br3 sorry for the context shift-- we've been consolidating issues from core repos into one place for easier maintenance)

**Comment by l1br3** [comment]
no worries mike

**Comment by sailsbot** [comment]
@sailsbot,@mikermcneil,@l1br3: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!
